__tests__/app.test.js

```diff
@@ -100,3 +100,39 @@ describe('api/articles/:article_id',()=>{
         })
     })
 })
+describe('/api/article',()=>{
+    test('GET:200 should return an array of article objects with the correct properties',()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body:{articles}})=>{
```

Comment:
It's crucial that we include a length check here on the response before going into the forEach as if the response we are getting back was an empty array our forEach would never run, meaning our expect statements never run and we get an auto pass.